### DISCUSSION TITLE
where is the container's file stored after `runc run MY_CONTAINER`?

### DISCUSSION THREAD

**Discussion by Zhenshan-Jin** [opencontainers/runc#3519] [Q&A] [answered]
I want to check the files created only by `runc run MY_CONTAINER` similar to the files in `/var/lib/docker/containers/MY_CONTAINER_ID`

**Comment by Zhenshan-Jin** [comment]
looks like it is under `/run/crun/MY_CONTAINER`

**Reply by cyphar** [reply]
That's for crun (https://github.com/containers/crun) which is a different project.

**Comment by cyphar** [comment, marked as answer]
`/var/run/runc/MY_CONTAINER`, though this can be changed with using the `--root` flag (and in the case of rootless containers, the root used is `$XDG_RUNTIME_DIR/runc`).
